fossology/fossology
# Feat/spasht

## Description

Added Agent Spasht With a Basic Configuration UI

### Changes

#### Registered Agent
#### Inserted Spasht Menu in the UI menu option
#### Added the feature o get Upload File Name

## How to test

Run `sudo make install` for the first time.
open an uploaded file.
Check spasht in micro menu options.
